common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -15,37 +15,35 @@
 import com.wynntils.mc.utils.CrashReportManager;
 import com.wynntils.mc.utils.McUtils;
 import com.wynntils.wc.utils.WynnUtils;
+import java.util.ArrayList;
+import java.util.EnumMap;
 import java.util.LinkedList;
 import java.util.List;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 /** Loads {@link Feature}s */
 public class FeatureRegistry {
-    private static final List<Feature> FEATURES = new LinkedList<>();
-    private static final List<Overlay> OVERLAYS = new LinkedList<>();
+    private static final List<Feature> features = new LinkedList<>();
```

Comment:
static final members are conventionally named in upper case. Why change this?

Reply:
I've always found lower case more preferable but I am fine with upper case. I will revert that change